app/pages/learn/01_tutorial/01_your-first-java-app/02_building-with-intellij-idea.md

```diff
@@ -0,0 +1,221 @@
+slug: learn/intellij-idea
+type: tutorial
+category: start
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+more_learning:
+  youtube:
+    - H_XxH66lm3U
+    - 70_B2DyM8mU
+toc:
+- Installing IntelliJ IDEA
+- Creating a new project
+- Writing and editing code
+- Running your application
+- Testing
+- Debugging
+- Refactoring code
+- Documenting code
+- Searching and navigating
+- Version Control
+- Evolving the project
+- More information
+description: "IntelliJ IDEA is the most used Java IDE and helps you develop your Maven and Gradle Java projects faster."
+last_update: 2024-03-12
+---
+
+An IDE (Integrated Development Environment) allows you to quickly build applications, by integrating a source-code editor with the ability to compile and run your code, as well as integration with any other tools you’ll need for software development, including build tools, testing and debugging tools, version control, databases, and so on. And finally, an IDE will let you search and navigate your codebase in ways your file system won’t.
+
+IntelliJ IDEA is the [most widely used IDE for Java developers](https://www.jetbrains.com/lp/devecosystem-2023/java/#java_ide). It has out-of-the-box integrations with all the tools a Java developer will need for software development, with no need to install any additional plugins to get started.
+IntelliJ IDEA is available in two editions:
+- **_IntelliJ IDEA Community Edition_** - free and open-source, licensed under Apache 2.0. It provides all the basic features for Java and Kotlin development.
+- **_IntelliJ IDEA Ultimate_** - commercial, distributed with a 30-day trial period. It provides additional tools and features for web and enterprise development.
+
+## Installing IntelliJ IDEA
+To install IntelliJ IDEA, download the version you want to use from the [IntelliJ IDEA website](https://www.jetbrains.com/idea/) and follow the instructions. For more information on installing IntelliJ IDEA on your OS, see [the documentation](https://www.jetbrains.com/help/idea/installation-guide.html#standalone).
+
+When you launch IntelliJ IDEA for the first time, you’ll see the **Welcome** screen. From here, you create a new project, open an existing project, or get a project from a version control system.
+
+[![Welcome screen](/assets/images/intellij-idea/welcome-screen.png)](/assets/images/intellij-idea/welcome-screen.png)
+
+There is no need to install any additional plugins to get started; the most common integrations are already available as part of the IDE or as bundled plugins. Of course, you can [customize the IDE](https://www.jetbrains.com/help/idea/run-for-the-first-time.html#select-ui-theme) further, or [install additional plugins](https://www.jetbrains.com/help/idea/run-for-the-first-time.html#additional_plugins) if you really want to.
+
+You will need to install a JDK, if you don’t already have one installed. You can take care of that when creating a project.
```

Comment:
It may be a good idea to link to https://dev.java/learn/getting-started/ for that.

Reply:
Good idea. Or more specifically, I'd link to: https://dev.java/learn/getting-started/#setting-up-jdk
Otoh, the whole point of using an IDE is that you can do many things from within the IDE, without having to switch tools (as is mentioned later about downloading the JDK).